schedule/yast/autoyast/autoyast_ext4.yaml

```diff
@@ -5,7 +5,10 @@ description: >
   with product defined in the profile.
 vars:
   AUTOYAST: autoyast_sle15/autoyast_ext4.xml
+  AUTOYAST_PREPARE_PROFILE: '1'
   DESKTOP: gnome
+  INSTALLATION_VALIDATION: autoyast/verify_ext4
```

Comment:
INSTALLONLY=1 is missing

Reply:
I am a bit confused about this variable because the description is "Indicates that test suite conducts only installation. Is recommended to be used for all jobs which create and publish images." So, this test publishes image but there is also validation following the installation.

Reply:
From POV is used in the lib/main_common.pm to take specific actions. but it speads all over there and i cant say where it could affect a process or not. for instance https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/main_common.pm#L631 or https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/main_common.pm#L659. i would not remove it for now. But maybe the other guys have a better idea about it.

Reply:
I removed all INSTALLONLY settings, as Rodion mentioned that it is used for scheduling and we no longer need it.